packages/appstore/src/components/wallet-action-button/wallet-action-button.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { localize } from '@deriv/translations';
+import { Text } from '@deriv/ui';
+import { Icon } from '@deriv/components';
+
+type TWalletActionButtonProps = {
+    size: 'small' | 'medium' | 'large';
+    icon: string;
+    label?: string;
+    onClick?: (e: any) => void;
```

Comment:
```suggestion
    onClick?: React.MouseEventHandler<HTMLButtonElement>;
```
@iman-fs Not sure about it, Please check 🙏🏻

Reply:
Yes. there is no problem with it. thanks :+1: